requirements/required.txt

```diff
@@ -2,6 +2,7 @@
 # standard library in Python >= 2.7
 argparse
 six>=1.9
+ordereddict ;python_version <= '2.6'       # JSON
```

Comment:
I would remove the JSON comment. This package might be necessary to address #3396 in a proper way.